Classes/foxPlayerInput.uc

```diff
@@ -27,6 +27,7 @@ var globalconfig float Desired43FOV;
 var globalconfig bool bCorrectZoomFOV;
 var globalconfig bool bCorrectMouseSensitivity;
 var globalconfig float Desired43MouseSensitivity;
+var globalconfig bool bCorrectWeaponFOV;

```

Comment:
This looks good, I would just move it up 2 lines to sit under "bCorrectZoomFOV", so that the FOV options are grouped together.